pkg/acsengine/transform/apimodel_merger.go

```diff
@@ -23,7 +23,7 @@ type APIModelValue struct {
 
 // MapValues converts an arraw of rwa ApiModel values (like ["masterProfile.count=4","linuxProfile.adminUsername=admin"]) to a map
 func MapValues(m map[string]APIModelValue, setFlagValues []string) {
-	if setFlagValues == nil || len(setFlagValues) == 0 {
+	if len(setFlagValues) == 0 {
```

Comment:
no... really, are you sure you're unset?

Reply:
`pkg/acsengine/transform/apimodel_merger.go:26:5:warning: should omit nil check; len() for nil slices is defined as zero (S1009) (gosimple)`.